tests/View/ViewBladeCompilerTest.php

```diff
@@ -814,4 +814,28 @@ public function testGetTagsProvider()
             ['(((', ')))'],
         ];
     }
+
+    /**
+     * @expectedException \RuntimeException
```

Comment:
The `BladeCompiler::compileStatements` checks that the parenthesis are well balanced before it get a chance to be passed down to any other `compile`, it's hard to give any good error messages down the chain...